tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/Serverless.cs

```diff
@@ -17,16 +15,9 @@ internal static class Serverless
 
         private static NativeCallTargetDefinition[] callTargetDefinitions = null;
 
-        internal static LambdaMetadata Metadata { get; private set; } = LambdaMetadata.Create();
-
-        internal static void SetMetadataTestsOnly(LambdaMetadata metadata)
-        {
-            Metadata = metadata;
-        }
-
-        internal static void InitIfNeeded()
+        internal static void InitIfNeeded(LambdaMetadata metadata)
```

Comment:
Not related but I feel we should change the CodeOwners file to automatically notify the serverless team when we chose those files.

Reply:
and that would be which github team? 🤔 (I'll do a new PR)

Reply:
You should ask them, I'm not sure